fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/CreateNamedStruct.java

```diff
@@ -66,14 +69,18 @@ public void checkLegalityBeforeTypeCoercion() {
                 throw new AnalysisException("named_struct only allows"
                         + " constant string parameter in odd position: " + this);
             } else {
-                String name = ((StringLikeLiteral) child(i)).getStringValue();
+                String name = ((StringLikeLiteral) child(i)).getStringValue().toLowerCase();
```

Comment:
current, struct is case sensitive, so here do case insensitive name conflict check is weird